envs/tensorflow-s390x-extra-deps-env.yaml

```diff
@@ -8,6 +8,15 @@ packages:
     git_tag: d8c73a73f29e835eb5dfc5e071b9564ea665e7fd
   - feedstock : https://github.com/AnacondaRecipes/promise-feedstock
     git_tag: a1855c8a4e3e2e597f612e4edd504bf1da81d91d
+  - feedstock : https://github.com/conda-forge/llvmdev-feedstock
+    git_tag : c546c92e4ca06051bf48bda7e97138f348924293
+    patches:
+      - feedstock-patches/llvm/0001-LLVM-fixed-with-old-compiler.patch
+  - feedstock : https://github.com/conda-forge/clangdev-feedstock
+    git_tag : 28f7809e7f4286b203af212a154f5a8327bd6fd6
+    patches :
+      - feedstock-patches/clang/0001-Fix-clang-build-with-GCC-11.patch
+      - feedstock-patches/clang/0002-Fix-clang-build-for-s390x.patch                          # [s390x]
```

Comment:
Is [0001-Build-fixed-with-old-compilers-and-new-libgcc-ng.patch](https://github.com/open-ce/open-ce/pull/693/files#diff-ea54dd905b6756b2b789310e04ddd77483a9cd81fb01461b7b3a3510a9adad65) not needed?

Reply:
That's great. Thanks @npanpaliya.